pkg/conn/grpc_test.go

```diff
@@ -0,0 +1,14 @@
+package conn
+
+import "testing"
+
+const (
+	endpoint = "localhost:9000"
+)
+
+func TestConnectTogRPC(t *testing.T) {
+	_, err := ConnectToSocket(endpoint)
+	if err != nil {
+		t.Fatal(err)
```

Comment:
If you're testing that the `ConnectToSocket` function returns an error. It is better to use assertion such as `assert.NotNil(t, err)` in a `t.Run`.